ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -32,6 +32,19 @@ RSkComponentTextInput::RSkComponentTextInput(const ShadowView &shadowView)
     ,cursor_({0,0}){
   RNS_LOG_DEBUG("called constructor");
   paragraph_ = nullptr;
+  cursorPaint_.setColor(RSkColorFromSharedColor(selectionColor_, SK_ColorBLUE));
```

Comment:
setColor should be with BLUE color here ? since selectionColor does not have any value assigned